manticore/core/cpu/arm.py

```diff
@@ -760,13 +773,7 @@ def BLX(cpu, dest):
         ## The `blx <label>` form of this instruction forces a state swap
```

Comment:
is the XXX/TODO comment above still accurate? since we do support thumb can it be rm'd?